configure.ac

```diff
@@ -1050,6 +1050,15 @@ fi
 AM_CONDITIONAL(HAVE_LIBIDN, [test "$have_libidn" = "yes"])
 
 # ------------------------------------------------------------------------------
+have_idn=no
+AC_ARG_ENABLE(idn, AS_HELP_STRING([--disable-idn], [disable optional IDN support]))
```

Comment:
This is not clear enough. We have `--disable-libidn` and `--disable-libidn2`, which are independent of this option. So saying that this "disables IDN support" is not true. What about "disable IDN when printing host names" ?